charts/develop/jdownloader2/SCALE/ix_values.yaml

```
@@ -0,0 +1,20 @@
+##
+# This file contains Values.yaml content that gets added to the output of questions.yaml
+# It's ONLY meant for content that the user is NOT expected to change.
+# Example: Everything under "image" is not included in questions.yaml but is included here.
+##
+
+image:
+  repository: jlesage/jdownloader-2
+  tag: v1.7.1
+  pullPolicy: IfNotPresent
+
+#All values here are set as the docker defaults.
+envTpl:
+  # Permissions Settings
+  USER_ID: "{{ .Values.env.PUID }}"
+  GROUP_ID: "{{ .Values.env.PGID }}"
+##
```

Try adding some whistespace for readability
```suggestion
  GROUP_ID: "{{ .Values.env.PGID }}"

##
```